openapi.yaml

```diff
@@ -2711,16 +2711,14 @@ components:
 
     AttemptResponse:
       type: object
-      required: [webhook_url, redirect_url, id, _links, status]
+      required: [webhook_url, redirect_url, id, _links, status, created_on, modified_on, response_codes]
       properties:
         id:
           type: string
         status:
           type: string
         redirect_url:
           type: string
-        webhook_url:
```

Comment:
can you please update the list attempts example as well?

Reply:
list and detail, they both comtain the webhook_url

Reply:
see `caca5` commit